src/EPR.SubsidiaryBulkUpload.Application/Services/CompaniesHousCsvProcessor.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Globalization;
+using CsvHelper;
+using CsvHelper.Configuration;
+using EPR.SubsidiaryBulkUpload.Application.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+
+namespace EPR.SubsidiaryBulkUpload.Application.Services;
+
+public class CompaniesHousCsvProcessor(
+    ILogger<CompaniesHousCsvProcessor> logger) : ICsvProcessor
+{
+    private readonly ILogger<CompaniesHousCsvProcessor> _logger = logger;
+
+    public async Task<int> ProcessStream(Stream stream)
```

Comment:
I don't think this method is being used. Can it be deleted?

Reply:
Leave this until we merge with the other branch code - the interface is used by both